ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionScopeField.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FormGroup, FormGroupProps, Radio } from '@patternfly/react-core';
+
+import { ScopeContext } from './utils';
+
+export type ExceptionScopeFieldProps = {
+    fieldId: FormGroupProps['fieldId'];
+    label: FormGroupProps['label'];
+    scopeContext: ScopeContext;
+};
+
+function ExceptionScopeField({ fieldId, label, scopeContext }: ExceptionScopeFieldProps) {
+    return (
+        <FormGroup fieldId={fieldId} label={label} isRequired>
+            {scopeContext === 'GLOBAL' && (
+                <Radio
+                    id="scope-global"
+                    name="scope-global"
+                    isChecked={false}
```

Comment:
This caught my eye in the picture.

When state has been implemented, will only radio option be selected and maybe disabled?

Reply:
Hmm good question. Currently this will be the only option selected, with no opportunity for the user to change it, but according to the mocks it should be in a disabled state instead.

Reply:
Updated in this PR, as the e2e tests needed adjustment as well.